baseball/src/components/Display.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+class Display extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Makes sense to have your Display component to hold all of the state and functions.